packages/instant/src/components/erc20_asset_amount_input.tsx

```diff
@@ -23,6 +23,7 @@ export interface ERC20AssetAmountInputProps {
     startingFontSizePx: number;
     fontColor?: ColorOption;
     isDisabled: boolean;
```

Comment:
Lets rename this prop into something more specific like `isInputDisabled`

Reply:
Done in b7eb2e8